analyzer/src/main/kotlin/managers/utils/NuGetSupport.kt

```diff
@@ -137,11 +137,16 @@ class NuGetSupport(
             val packageData = registrationsBaseUrls.firstNotNullOfOrNull { getPackageData(id, it) }
 
             if (packageData == null) {
+                val hint = " Support for non-fixed versions, like '${id.version}', is not implemented".takeIf {
```

Comment:
"like" sounds like an example, but in fact you're checking exactly that version, so I'd simply say " Support for non-fixed version '${id.version}' is not implemented." (also note the added trailing dot at the end of the sentence).